products/sqlite.md

```diff
@@ -0,0 +1,53 @@
+---
+title: SQLite
+category: db
+iconSlug: sqlite
+permalink: /sqlite
+alternate_urls:
+-   /sqlite3
+versionCommand: sqlite3 --version
+releasePolicyLink: https://www.sqlite.org/support.html
+changelogTemplate: "https://www.sqlite.org/changes.html#version_{{'__LATEST__'|replace:'.','_'}}"
+releaseDateColumn: true
+activeSupportColumn: false
+eolColumn: Support Status
+
+# This git mirror only contains versions from 3.6.10.
+auto:
+-   git: https://github.com/sqlite/sqlite.git
+    regex: '^version-(?<major>\d+)\.(?<minor>\d+)(\.(?<patch>\d+))?$'
+
+releases:
+-   releaseCycle: "3"
+    eol: false
+    latest: "3.40.1"
+    latestReleaseDate: 2022-12-28
+    releaseDate: 2004-06-18
+
+-   releaseCycle: "2"
+    releaseDate: 2001-09-28
+    # 3.0.7 release date, because it is the first stable release in the 3.x line
+    eol: 2004-09-18
+    latest: "2.8.17"
+    latestReleaseDate: 2005-12-19
+
+-   releaseCycle: "1"
+    releaseDate: 2000-08-17
+    eol: 2001-09-28
+    latest: "1.0.32"
+    latestReleaseDate: 2001-07-23
+
+---
+
+> SQLite is an in-process library that implements a self-contained, serverless, zero-configuration,
+> transactional SQL database engine. The code for SQLite is in the public domain and is thus free
```

Comment:
` and is thus` looks like a typo here

Reply:
LGTM : https://www.merriam-webster.com/dictionary/thus.